postprocessing/janus-pp-rec.c

```diff
@@ -709,7 +709,12 @@ int main(int argc, char *argv[])
 			continue;
 		}
 		/* Only read RTP header */
-		bytes = fread(prebuffer, sizeof(char), len > 24 ? 24: len, file);
+		uint16_t rtp_header_len = len > 24 ? 24: len;
+		bytes = fread(prebuffer, sizeof(char), rtp_header_len, file);
+		if (bytes < rtp_header_len) {
+			JANUS_LOG(LOG_WARN, "Missing RTP packet header data (%i instead %i)\n", bytes, rtp_header_len);
```

Comment:
Since `rtp_header_len` is defined as `uint16_t`, the printf format should be `%"SCNu16"`; for bytes you might want to use `%d` since we don't use `%i` anywhere else (so just a matter of consistency). As a side note, considering it's used in all iterations of the loop, it might make sense to define the `rtp_header_len` variable just once initially, as we do for `bytes`.

Reply:
all done !